apps/data/database/initdb.sql

```diff
@@ -90,4 +90,27 @@ CREATE TABLE user_clicks (
     user_id INTEGER NOT NULL REFERENCES users(user_id) ON DELETE CASCADE,
     listing_id INTEGER NOT NULL REFERENCES listings(listing_id),
     created_at TIMESTAMP NOT NULL DEFAULT NOW()
+);
+
+CREATE TABLE organizations (
+    organization_id SERIAL PRIMARY KEY,
+    name VARCHAR NOT NULL,
+    logo_url TEXT,
+    donated DECIMAL(10, 2),
+    receiving BOOLEAN
+);
+
+CREATE TABLE charities (
+    charity_id SERIAL PRIMARY KEY,
+    name VARCHAR NOT NULL,
+    description TEXT,
+    start_date DATE,
+    end_date DATE,
+    image_url TEXT
+);
+
+CREATE TABLE charity_organization (
+    charity_id INTEGER NOT NULL REFERENCES charities(charity_id) ON DELETE CASCADE,
+    organization_id INTEGER NOT NULL REFERENCES organizations(organization_id) ON DELETE CASCADE,
+    PRIMARY KEY (charity_id, organization_id)
```

Comment:
This could just be done by having a foreign key charity_id on organizations since each organization will only be linked to one charity